src/lib/holocene/navigation/_nav-container.svelte

```diff
@@ -32,7 +32,7 @@
       style="top: 52px;"
       on:click={toggleNav}
     >
-      <Icon name={$navOpen ? 'nav-collapse' : 'nav-expand'} />
+      <Icon name={$navOpen ? 'chevron-left' : 'chevron-right'} />
```

Comment:
Wondering if we want to remove `nav-collapse` and `nav-expand` icons to prevent any future confusion.

Reply:
Great idea, done